privacy-protections-popup/privacy-protections-popup-impl/src/main/java/com/duckduckgo/privacyprotectionspopup/impl/PrivacyProtectionsPopupPixelName.kt

```diff
@@ -70,6 +70,34 @@ enum class PrivacyProtectionsPopupPixelName(
         pixelName = "m_privacy_protections_popup_page_refresh_on_possible_breakage_d",
         type = DAILY,
     ),
+    PRIVACY_DASHBOARD_LAUNCHED_UNIQUE(
+        pixelName = "m_privacy_protections_popup_dashboard_launched_u",
+        type = UNIQUE,
+    ),
+    PRIVACY_DASHBOARD_ALLOWLIST_ADD_UNIQUE(
+        pixelName = "m_privacy_protections_popup_dashboard_allowlist_add_u",
+        type = UNIQUE,
+    ),
+    PRIVACY_DASHBOARD_ALLOWLIST_REMOVE_UNIQUE(
+        pixelName = "m_privacy_protections_popup__dashboard_allowlist_remove_u",
```

Comment:
You have a double "_" here.
I assume the pixel name should be `m_privacy_protections_popup_dashboard_allowlist_remove_u`